src/metalnx-web/src/main/java/com/emc/metalnx/controller/UploadController.java

```diff
@@ -110,6 +114,16 @@
 		String resources = multipartRequest.getParameter("resources");
 		String resourcesToUpload = multipartRequest.getParameter("resourcesToUpload");
 		String destPath = multipartRequest.getParameter("uploadDestinationPath");
+		
+		if ("undefined".equalsIgnoreCase(resourcesToUpload)) {
+		    // TODO Checking if the default resource is set should probably happen on
+		    // startup of the web application.
```

Comment:
did we want to remove this comment?

maybe we make it true in the future? that is, make the server report when a required option has not been defined.

Reply:
yes, I'd say that's part of making #219 real

Reply:
removed comment.